### ISSUE TITLE
TPO/tp to player.

### ISSUE THREAD

**Issue by noopster1** [SavageLabs/SavageSkyblock-OLD#27]
I am experiencing a bug when using the tp/tpo command. When tping, it will not show the border and it will not allow you to break blocks even when in bypass mode. However, when doing /is visit <player> it will allow me to break blocks and see the border. Also when tping to a player (not the player island) it will show the title, but it shows another users island rather than their username.

**To Reproduce**
1: Go to a different world other than the skyblock world.
2. Do /tp /tpo to a player on the server, and you will see that you can not break blocks, see border & a different title username.

**Expected behavior**
The expected behavior is to tp to a player and still be able to break blocks (without bypass mode), see the border & title the correct username.

**Screenshots**
Sorry! I do not have screenshots.

**Server Version (please complete the following information):**
 - Spigot 1.8.8 (running via version)
 - I use the latest version of EpicSkyblock as of June 19, 2019

**Additional context**
No additional context.

**Comment by prosavage** [comment]
Yeah this is being caused by manually applying the event, I will implement something using the teleport event, we have to fix a lot of the code before we get to it.

**Comment by noopster1** [comment]
Will you also be adding PlaceHolderAPI support?
